build.gradle

```diff
@@ -33,12 +33,20 @@ task start(type: NpmTask, dependsOn: 'npmInstall') {
     args = ['run', 'start'] // corresponds with script @ package.json/scripts.start
 }
 
-task test(type: NpmTask, dependsOn: 'npmInstall') {
+task testChrome(type: NpmTask, dependsOn: 'npmInstall') {
     group = 'verification'
-    description = 'Run ng test'
-    args = ['run', 'test'] // corresponds with script @ package.json/scripts.test
+    description = 'Run ng test --browser ChromeHeadless --single-run'
+    args = ['run', 'testCIChrome'] // corresponds with script @ package.json/scripts.testCIChrome
 }
 
+task testFirefox(type: NpmTask, dependsOn: 'npmInstall') {
+    group = 'verification'
+    description = 'Run ng test --browser Firefox --single-run'
+    args = ['run', 'testCIFirefox'] // corresponds with script @ package.json/scripts.testCIFirefox
+}
+
+task test(dependsOn: 'testFirefox')
```

Comment:
not sure if we need this, Jenkinsfile should call 'testFirefox' - would be clearer